Automattic/jetpack
# Instant Search: Use static, stable names for async chunks

Fixes #24067.

#### Changes proposed in this Pull Request:


Defines static and unchanging names for chunk files generated by Webpack.





#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your Jetpack (or Jurassic Ninja) site with a Jetpack Search subscription.
* Ensure that Instant Search has been enabled and navigate to `/?s=post`.
* Ensure that the page loads as expected.
* Check the network panel and ensure that a chunk file named `jp-search.defaultVendors.js` is loaded.
